Library/Formula/lua.rb

```diff
@@ -114,7 +117,7 @@ def install
   def pc_file; <<-EOS.undent
     V= 5.2
     R= 5.2.4
-    prefix=#{HOMEBREW_PREFIX}
+    prefix=#{prefix}
```

Comment:
My motivation for using `prefix` instead of `opt_prefix` was consistency with basically every other formula that exists today, as the value in the `.pc` file usually comes from `./configure --prefix=`. (I remembered the discussion about not wanting to mass-rewrite `.pc` files to `opt`.)